src/components/avatar/Avatar.tsx

```diff
@@ -2,7 +2,7 @@ import * as React from 'react';
 import classNames from 'classnames';
 import { Text, Icon } from '@/index';
 import { BaseProps, extractBaseProps } from '@/utils/types';
-import { TooltipProps } from '@/index.type';
+// import { TooltipProps } from '@/index.type';
 import { AccentAppearance } from '@/common.type';
 
 export type AvatarSize = 'regular' | 'tiny';
```

Comment:
Remove Avatar changes

Reply:
We need to make tooltip changes for build to pass since tooltip is yet to be migrated to next version